### DISCUSSION TITLE
INAV 5.1 PIDs for 7 inch quad

### DISCUSSION THREAD

**Discussion by Diver6691** [iNavFlight/inav#8377] [Q&A]
Hi,
New in INAV.
Can anyone share the PIDs and filters for long range smooth 7 inch flight , so I can have a starting point ?
Motors are 2807
Thanks

**Comment by JulioCesarMatias** [comment]
https://github.com/iNavFlight/inav/wiki/PID-for-various-aircrafts

**Reply by Diver6691** [reply]
Thanks

**Comment by JulioCesarMatias** [comment]
https://github.com/iNavFlight/inav/wiki/Default-values-for-different-type-of-aircrafts
